packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -256,6 +259,32 @@ export default function ProjectNav({
   const isCloudManager = current?.planType === PlanType.Cloud;
   const isLoading = !!current;
 
+  const currentProjectId = current?.projectId;
+
+  function fetchAlertsCount() {
+    return fetch(
+      `${hosts.cloud}/user/shared/alerts/project/${currentProjectId}`,
```

Comment:
Given that `currentProjectId` is possibly undefined (based on the optional chaining operator in its declaration), we might want to check that it's defined before fetching.

Reply:
I tricked you! Look at the enabled value for usePoller! Won't run unless it's set.

Reply:
Ahah! Missed that. Looks good 😁